en/Newtons_law_of_gravitation.md

```diff
@@ -0,0 +1,50 @@
+# Explanation of Newtons Law of Gravitation
+
+
+Newtons Law of Gravitation explains the nature of the `Gravitational force` which is a force of attraction between two objects having definite mass and at a distance
+which is greater than 0.The law states that the force of gravitation is inversly proportional to the square of distance between the two distinct object.The force
+applied on object 1 is equal in magnitude to the force applied on object 2 but opposite in direction. 
+
+**Formula of Newtons Law of Gravitation:**
+
+F = (G * M_1 * M_2) / (r)^2
+where
+G= Universal gravitational Constant
+M_1 = Mass of Distinct Object 1
+M_2 = Mass of Distinct Object 2
+r = Distance between Distinct Object 1 and Distinct Object 2
+<p align="center">
+    <img width="60%" src="https://img.favpng.com/3/4/9/newton-s-law-of-universal-gravitation-physical-body-force-gravitational-constant-png-favpng-4mLrMUz7YmmE2btgGesXjgN3B.jpg">
+</p>  
+
+**Method of Using the Newtons Law of Gravitation:**
+
+Using the above given fromula, we just have to place the variables in their respective places:
+- **G (Universal Gravitational Constant)**=`6.6743e-11`. Unit of **G** is N(m^2)/(Kg^2).
+- **M_1**=`Mass of Distinct object 1 `. Unit of **M_1** is Kg.
+- **M_2**=`Mass of Distinct object 2 `. Unit of **M_2** is Kg.
+- **r**=`Distance between distinct object 1 and distinct object 2`. Unit of **r** is m(Meter).
+
+# Assumptions
+
+- The 2 distinct objects must not be weightless .
+- Their centers of gravity must not be one i.e they must have their centers of gravity away from each other.
+- This formula can only be used to calculate the Gravitation Force for 2 objects in a instance.
+
+
+# Uses 
+
+This law is used to determine the Force of attraction between 2 distinct Objects/Bodies , in order to calculate the rate of acceleration of either body towards the 
+other body with one being at a complete stop .
+
+# Example:-
+
+Let M_1 be a Iron ball of mass `5 * 10^6 Kg` and M_2 be a Copper ball of mass `10^4 Kg` and the seperation between the 2 objects **r**=100m
+Then the Graviational force between the 2 distinct objects is = `0.0003337 N`  or  `3.337 * 10^-4 N` 
+
+# Source
+- [Wikipedia](https://en.wikipedia.org/wiki/Newton%27s_law_of_universal_gravitation)
+- [Youtube](https://www.youtube.com/watch?v=ruBfXIVSYZ8)
+
+# Implementation
+- [newtos_law_of_gravitations](https://github.com/TheAlgorithms/Algorithms-Explanation/blob/master/en/Basic%20Math/Arithmetic%20Progressions.md)
```

Comment:
The implementations header is used if there are any implementations of this algorithm in some of [The Algorithms repositories](https://github.com/orgs/TheAlgorithms/repositories). If there are none, feel free to remove this part.

Reply:
I have made the suggested changes, would it now be possible to accept this